content/docs/testing-farm.md

```diff
@@ -83,13 +83,29 @@ in the pull request:
 
 ## Creating Tests
 
-The easiest way to get started is to use the [tmt][tmt] tool
+The easiest way to get started with defining tests is to use the [tmt][tmt] tool
 which will help you with the setup.
 Please follow [Fedora tmt page][fedora-tmt].
```

Comment:
Maybe we could point users to the [guide](https://tmt.readthedocs.io/en/latest/guide.html)? It give a nice step-by-step introduction.